helpers/contains.js

```diff
@@ -21,6 +21,28 @@ const factory = () => {
     };
 };
 
+const containerIncludesValue = (container, value) => {
+    switch (typeof container) {
+        case 'string':
+            return container.includes(value);
+        case 'object':
+            if (container) {
+                if  (Array.isArray(container)) {
+                    return container.includes(value);
+                }
+
+                // Check if the supplied value matches a top level value in the container object
+                return Object.values(container).some(objValue => {
```

Comment:
I would have liked for us to keep this simple like `Object.values(object1).includes(value);` but I guess if lodash is supporting complex type this make is super difficult for us